portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+                }));
+            })
+            .finally(() => {
+                setLoading(false);
+            });
+    }
+
+    useEffect(() => {
+        fetchPolicies();
+        if (!isCreateNew && policyID){
+            fetchGlobalPolicyByID();
+        }
+    }, []); 
+
+    /**
+     * A Context Operation for Policy Panel UI.
+     * Triggers as we click delete icon in a drag`n`droped the policy.
+     * @param {string} uuid - Operation uuid.
+     * @param {string} currentFlow - Which flow needs to be udpated: request, response or fault.
+     */
+    const deleteGlobalOperation = (uuid: string, currentFlow: string) => {
+        const newGlobalLevelPolicies: any = cloneDeep(globalLevelPolicies);
+        const index = newGlobalLevelPolicies[currentFlow].map((p: any) => p.uuid).indexOf(uuid);
+        newGlobalLevelPolicies[currentFlow].splice(index, 1);
+        setGlobalLevelPolicies(newGlobalLevelPolicies);
+    }
+
+    /**
+     * A Context Operation for Policy Panel UI.
+     * Function to rearrange the API Operation ordering.
+     * @param {string} oldIndex - Original index of the policy.
+     * @param {string} newIndex - New index of the policy.
+     * @param {string} currentFlow - Which flow needs to be udpated: request, response or fault.
+     */
+    const rearrangeGlobalOperations = (
+        oldIndex: number, newIndex: number, currentFlow: string,
+    ) => { 
+        const newAPIPolicies: any = cloneDeep(globalLevelPolicies);
+        const policyArray = newAPIPolicies[currentFlow];
+        newAPIPolicies[currentFlow] = arrayMove(policyArray, oldIndex, newIndex);
+        setGlobalLevelPolicies(newAPIPolicies);   
+    };
+
+    /**
+     * A Context Operation for Policy Panel UI.
+     * Triggers as we saved a drag`n`droped policy or edit a already dragged one.
+     * @param {any} updatedOperation - Saved info as parameters: {headerName: <>, headerValue: <>}, policyId: <>, etc.
+     * @param {string} currentFlow - Folow request/response/fault.
+     */
+    const updateGlobalOperations = (
+        updatedOperation: any, currentFlow: string,
+    ) => {
+        const newGlobalLevelPolicies: any = cloneDeep(globalLevelPolicies);
+        /**
+        * Check whether the policy operation already exists.
+        */
+        const flowPolicy = (newGlobalLevelPolicies)[currentFlow].find(
+            (p: any) =>
+                p.policyId === updatedOperation.policyId &&
+                p.uuid === updatedOperation.uuid,
+        );
+        
+        if (flowPolicy) {
+            /**
+            * Edit the already dragged and dropped policy.
+            */
+            flowPolicy.parameters = { ...updatedOperation.parameters };
+        } else {
+            /**
+            * Save the newly dragged and dropped policy.
+            */
+            const uuid = uuidv4();
+            (newGlobalLevelPolicies)[currentFlow].push({ ...updatedOperation, uuid }
+            );
```

Comment:
Formatting

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)